ui/raidboss/data/04-sb/ultimate/unending_coil_ultimate.js

```diff
@@ -1837,39 +1832,29 @@
         'Earth Shaker': 'Erdstoß',
         'Exaflare': 'Exaflare',
         'Fellruin Trio': 'Untergangs-Trio',
-        'Fireball(?! Soak)': 'Feuerball',
         'Fireball Soak': 'Feuerball aufsaugen',
-        'Flames Of Rebirth': 'Flammen der Wiedergeburt',
+        'Fireball(?! Soak)': 'Feuerball',
         'Flare Breath': 'Flare-Atem',
         'Flatten': 'Einebnen',
         'Generate': 'Formung',
         'Gigaflare': 'Gigaflare',
         'Grand Octet': 'Großes Oktett',
-        'Hatch': 'Austritt',
         'Heavensfall Trio': 'Himmelssturz-Trio',
         'Heavensfall(?! )': 'Himmelssturz',
         'Hypernova': 'Supernova',
-        'Iceball': 'Eisball',
-        'Iron Chariot': 'Eiserner Streitwagen',
         'Liquid Hell': 'Höllenschmelze',
-        'Lunar Dive': 'Lunarer Sturz',
-        'Lunar Dynamo': 'Lunarer Dynamo',
-        '(?<! )Marker(?!\\w)': 'Marker',
         'Markers appear': 'Marker escheinen',
         'Megaflare': 'Megaflare',
-        'Megaflare Dive': 'Megaflare-Sturz',
         'Meteor Stream': 'Meteorflug',
         'Meteor/Dive or Dive/Beam': 'Meteor/Sturzflug oder Sturzflug/Strahl',
         'Morn Afah': 'Morn Afah',
         'Nael Dive': 'Nael Sturzflug',
         'Nael Marker': 'Nael Marker',
-        'Pepperoni': 'Pepperoni', // FIXME
+        'Pepperoni': 'Salami',
```

Comment:
💯

Reply:
in german this is the valid translation for it and when you encountered it first in t13 it makes sence.